microsetta_interface/implementation.py

```diff
@@ -813,25 +812,78 @@ def get_create_human_source(*, account_id=None):
     if has_error:
         return consent_output
 
+    form_type = "source"
+
     return _render_with_defaults(
         'new_participant.jinja2',
         tl=consent_output,
         post_url=post_url,
         duplicate_email_check_url=duplicate_email_check_url,
         home_url=home_url,
+        form_type=form_type,
         language_tag=session_locale())
 
 
 @prerequisite([ACCT_PREREQS_MET])
 def post_create_human_source(*, account_id=None, body=None):
```

Comment:
It looks like this function's primary purpose is now posting consent signatures, only some of which create a source. Can we update the function name (and corresponding API endpoint) to better reflect what it's doing? Additionally, it would probably be good to encapsulate the source creation code in an internal function (e.g. _create_human_source()) to keep code logically grouped.

Reply:
@cassidysymons The purpose of this function is to render consent page for source creation, data re-consent, and biospecimen consent. We agree that the name of this function might not be appropriate in the existing scenario. Would it be appropriate to rename this function and the route to render_consent_page?

Reply:
We try to keep the convention of starting function names with the HTTP method, so post_consent or something similar would be appropriate.

Reply:
ok will change to get_consent_form?

Reply:
Can you please make this change?

Reply:
Changes done!